stage1/gc/gc.go

```diff
@@ -96,7 +102,8 @@ func gcNetworking(podID *types.UUID) error {
 	case err == nil:
 		n.Teardown(flavor, debug)
 	case os.IsNotExist(err):
-		// probably ran with --net=host
+		// either ran with --net=host, or failed during setup
+		_ = networking.CleanupGarbage(".", podID)
```

Comment:
let's not specifically have this function return an error then just ignore it - can we log it at least?